src/NuGet.Clients/NuGet.CommandLine/Commands/DownloadCommandBase.cs

```diff
@@ -16,7 +19,7 @@ public abstract class DownloadCommandBase : Command
         private readonly IPackageRepository _cacheRepository;
         private readonly List<string> _sources = new List<string>();
 
-        protected PackageSaveModes EffectivePackageSaveMode { get; set; }
+        protected PackageSaveMode EffectivePackageSaveMode { get; set; }
```

Comment:
The value is either `Files | Nupkg` or `Files | Nupkg | Nuspec` depending on v2 \ v3 restore.